src/app/app.component.scss

```diff
@@ -1,10 +1,14 @@
 @import "../scss/variables";
 
 .footer {
-  position: fixed;
+  position: relative;
```

Comment:
Hi @developerfred - I think this still needs to be `position: fixed;` to ensure the footer stays at the bottom when there are not many messages on the page.

The `padding-bottom` will resolve the button error.